cmd/proxy/actions/sumdb.go

```diff
@@ -0,0 +1,36 @@
+package actions
+
+import (
+	"net/http"
+	"net/http/httputil"
+	"net/url"
+	"path"
+	"strings"
+)
+
+func sumdbPoxy(url *url.URL, nosumPatterns []string) http.Handler {
```

Comment:
`s/sumdbPoxy/sumdbProxy`

We can totally do that in a follow-up though, just a typo / nitpick